Sources/Sentry/SentryHub.m

```diff
@@ -515,7 +516,7 @@ - (void)configureScope:(void (^)(SentryScope *scope))callback
  */
 - (BOOL)isIntegrationInstalled:(Class)integrationClass
 {
-    for (id<SentryIntegrationProtocol> item in SentrySDK.currentHub.installedIntegrations) {
+    for (id<SentryIntegrationProtocol> item in self.installedIntegrations) {
```

Comment:
Is there a reason this went via `SentrySDK.currentHub`?

Reply:
Im not sure about this, because this is older than me, but you can't have integrations per hub, integrations are global, this is the only way we know the integration is running. 
That being said, we also dont support multiple hubs, so, Im fine with `self`.